lib/dockerize_rails/docker_commands/docker_start.rb

```diff
@@ -0,0 +1,106 @@
+module DockerizeRails
+  module DockerCommands
+    module DockerStart
+      def self.start_rails
+        services = DockerHelpers.services_from_docker_compose
+        begin
+          container = Docker::Container.get(DockerHelpers.get_name(:rails, :container))
+          if container.info['State']['Running']
+            puts " ==> Container >#{DockerHelpers.get_name(:rails, :container)}< already running.".yellow
+          else
+            docker_start_container(container, services['rails'], :rails)
+            puts " ==> Container >#{DockerHelpers.get_name(:rails, :container)}< started successfully".green
+          end
+          0
+        rescue Docker::Error::NotFoundError
+          services.key?('rails') && docker_start(services['rails'], :rails)
+          puts " ==> Container >#{DockerHelpers.get_name(:rails, :container)}< started successfully".green
+          0
+        rescue Docker::Error::NotFoundError => exception
+          puts
+          puts exception.to_s.red
+          puts
+          1
+        end
+      end
+
+      def self.start_mysql
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for start_mysql is too high. [30.56/20] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Method has too many lines. [21/15] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)

<!-- Comment by [SideCI](https://sideci.com) -->